test/parallel/test-fs-promises-writefile.js

```diff
@@ -35,7 +35,15 @@ async function doRead() {
   assert.deepStrictEqual(buf, data);
 }
 
+async function doReadWithEncoding() {
+  const data = await fsPromises.readFile(dest, 'utf-8');
+  const syncData = fs.readFileSync(dest, 'utf-8');
+  assert.strictEqual(typeof data, 'string');
+  assert.deepStrictEqual(data, syncData);
```

Comment:
`assert.strictEqual()` was probably better here, but I've already landed this and it's not a biggie so I will not force push.

Reply:
@lpinca why?

Reply:
It's a string no? No need it to be deep.